### PR TITLE
main: show Unix Socket server network output

### PR DESCRIPTION
**Pull request by breznak**
- [x] also use UnixSocket as NETWORK_OUTPUT where available
  - [ ] add check for supported system
- [ ] fix error

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -196,6 +196,10 @@ static bool mainloop_1(void) {
 
 	// And also send them via UDP. This is fast, as it doesn't care what is on the other side.
 	caerOutputNetUDP(9, 4, polarity, frame, imu, special);
+
+        // on UNIX-like systems, we can use unix_socket
+        //TODO add check for OS and enable only where supported
```
Comment (breznak) [inline main.c]: needs to add a check for supported OS/libs

**Comment by breznak** [comment]
Also, a crash or termination (ctrl^C) leaves the socket file (/tmp/socket) and a restart of the server misbehaves until the file is manually removed. 

- either the server should override the file
- or warn and cease to function (but not sending some warnings like
```
Total events/second:        253 - Valid events/second:        2532016-10-06 17:13:28 (TZ+0200): CRITICAL: 11-UnixSocketServerOutput: Could not bind local Unix socket. Error: 98.
Total events/second:        253 - Valid events/second:        2532016-10-06 17:13:28 (TZ+0200): CRITICAL: 11-UnixSocketServerOutput: Could not bind local Unix socket. Error: 98
```

Also, the err behavior looks similar to #27

**Comment by llongi** [comment]
Possible, only TCP has been extensively tested. I'm now reworking this as part of the Windows support, so that there we have Unix Sockets on Unix platforms and named Pipes on Windows, so no OS check will be needed. And I'll extensively test all the network combinations at that point. :-)

**Comment by breznak** [comment]
Thank you for the info here and elsewhere, I'm looking forward for the networking goodies during the next week! :+1: 

For now I can prototy with TCP.

**Pull request closed without merging**